### ISSUE TITLE
Update User Dashboard to display Their Points

### ISSUE THREAD

**Issue by crookedneighbor** [Commander-Spellbook/website-v2#196]
Depends on #194 

Acceptance Criteria:

- [ ] update dashboard to show total points the user has accumulated
- [ ] provide ui to show history of point awards

**Comment by crookedneighbor** [comment]
No longer relevant as the backend system is happening in a separate app now.
